test/WebSites/BasicWebSite/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ViewFeatures.Internal;
+
+namespace BasicWebSite.Filters
+{
+    public class RedirectAntiforgeryValidationFailedResultFilter : IAsyncAlwaysRunResultFilter
```

Comment:
Consider making this sync: `IAlwaysRunResultFilter` since you aren't doing any async work in this

Reply:
I forgot that this makes an actual difference for filters. Changed! :+1: